examples/scv_et_rework.py

```diff
@@ -64,8 +62,11 @@ def scv_data_download_for_year(year, openet_collection=r"OpenET/ENSEMBLE/CONUS/G
 							}
 
 
-def download_updated_vw_et_images_by_year(download_folder=r"D:\vw_et_update_2023",
-											field_boundaries=field_boundaries_by_year) -> None:
+def download_updated_vw_et_images_by_year(download_folder: str = r"D:\vw_et_update_2023",
+											field_boundaries: Optional[dict[str: str]] = None) -> None:
+	if field_boundaries is None:
```

Comment:
Hmmm, I think we should undo this - since it's an example, I think it doesn't need to be robust to that sort of issue and we can safely provide that dictionary as the default